plothist/plotters.py

```diff
@@ -76,6 +132,33 @@ def update_variable_registry(
     variable_key, x_min, x_max, path="./variable_registry.yaml"
 ):
     # TODO: extend updating function
+    """
+    Update the range parameter for a variable in the variable registry file.
+
+    Parameters
+    ----------
+    variable_key : str
+        The key identifier of the variable to update in the registry.
+    x_min : float
+        The new minimum value for the range of the variable.
+    x_max : float
+        The new maximum value for the range of the variable.
+    path : str, optional
+        The path to the variable registry file (default is "./variable_registry.yaml").
+
+    Returns
+    -------
+    None
+
+    Raises
+    ------
+    RuntimeError
+        If the variable registry file does not exist, and the default path is used.
+
+    See Also
+    --------
+    create_variable_registry
+    """
     if not os.path.exists(path):
         if path == "./variable_registry.yaml":
```

Comment:
Weird behavior. We should always raise a RuntimeError if the registry does not exist

Reply:
[same](https://github.com/cyrraz/plothist/pull/3#discussion_r1220610562)